src/main/java/Duke.java

```diff
@@ -54,19 +54,20 @@ public static void main(String[] args) {
                     }
                     System.out.println("    ____________________________________________________________");
 
-                }else if (userInput.contains("delete")){
+                } else if (userInput.contains("delete")){
```

Comment:
You can use userInput.substring(userInput.indexOf(" ")) to obtain the number, then you would not need to specify the exact number based on the action.